.changeset/shy-dolphins-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-ag/meteor-admin-sdk": patch
+---
+
+Add title for Criteria for debugging
```

Comment:
generated by changeset add, don't know if I should merge 2 changesets into one manually?

Reply:
Maybe so.